tests/epyccel/test_tuples.py

```diff
@@ -22,8 +22,6 @@ def is_func_with_0_args(f):
 
 failing_tests = {
         'homogeneous_tuple_string':"Can't save a list of strings (#459)",
-        'tuple_homogeneous_return':"Can't return a tuple",
-        'tuple_inhomogeneous_return':"Can't return a tuple",
         'tuple_visitation_inhomogeneous':"Can't iterate over an inhomogeneous tuple",
         'tuple_homogeneous_string':"Can't save a list of strings (#459)",
```

Comment:
I have just noticed that the we have two similar keys in this dictionary: `'homogeneous_tuple_string'` and `'tuple_homogeneous_string'`.

Reply:
Huh I never noticed this. I don't know if it is relevant to this PR?
It looks like they are 2 very similar tests:
https://github.com/pyccel/pyccel/blob/50bbafac029d1e8f5bbd67b0852fa39b6e4d9bda/tests/epyccel/modules/tuples.py#L97-L100
https://github.com/pyccel/pyccel/blob/50bbafac029d1e8f5bbd67b0852fa39b6e4d9bda/tests/epyccel/modules/tuples.py#L462-L465

There don't seem to be any clear naming conventions in this file

Reply:
They don't test the same thing though. `homogeneous_tuple_string` tests the creation of a homogeneous tuple of strings.
`tuple_homogeneous_string` tests the use of the `tuple()` function on a homogeneous tuple of strings.

Reply:
Subtle difference, but important nonetheless